packages/rancher-alerting/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Alerting Drivers
-  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.26.0-0'
```

Comment:
Have we tested this chart on k8s 1.25 cluster? You could test it by provisioning an rke1 k8s 1.25 cluster on v2.7-head setup 

We will need to make changes is PSP's are used as mentioned in this ticket for supporting k8s 1.25 https://github.com/rancher/rancher/issues/39848

Reply:
tested installing the chart on 1.25 and applied the psp related changes as well. @prachidamle